src/main/kotlin/no/nav/syfo/service/VarselBusService.kt

```diff
@@ -41,6 +42,18 @@ class VarselBusService(
         }
     }
 
+    fun processVarselHendelseAsTMSEvent(event: EsyfovarselHendelse) {
```

Comment:
Hvis man leser bare denne filen, er det litt vanskelig å kjenne hva TMS i metodenavn står for. 
Ett forslag `processVarselHendelseAsMinSideMicrofrontendEvent` kan vurderes?

Reply:
Fikset